Add upstream option of browse sub-commnad

yujideveloper opened the pull request.

yujideveloper committed:
Add upstream option of browse sub-commnad

f440 commented:
It's almost good idea, but I want to browse any remote tracked repositories, like this:

```
$ git remote
origin
upstream
bob
alice
$ sah browse upstream #=> open upstream repository
$ sah browse bob #=> open bob's repository
```

Are you sure about this change? https://github.com/f440/sah/compare/master...browse-any-remotes

yujideveloper commented:
Thanks for review.

It's good idea.
However, I got message(e.x. `fatal: Not a git repository (or any of the parent directories): .git`) when run `sah browse REPO` in unmanaged directory of git.
So, I think that we would be better to use `Open3.capture3('git remote').first.split` instead of `%x(git remote).split`.

f440 commented:
> So, I think that we would be better to use Open3.capture3('git remote').first.split instead of %x(git remote).split.

Your indication is merged in mainline and I shipped new release including this feature.

Thanks.
